### ISSUE TITLE
Conflict with allowBackup and supportsRTL in manifest file.

### ISSUE THREAD

**Issue by maolmhuire** [myinnos/AlphabetIndex-Fast-Scroll-RecyclerView#33]
For some applications that have allowBackup and/or supportsRtl set to false we receive the following error when trying to use your library:

```
 Manifest merger failed : Attribute application@allowBackup value=(false) from [:corevero] AndroidManifest.xml:12:9-36
  	is also present at [com.github.myinnos:AlphabetIndex-Fast-Scroll-RecyclerView:1.0.8] AndroidManifest.xml:12:9-35 value=(true).
  	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at manifestMerger3423917709392243790.xml:7:5-9:19 to override.
```

Would it be possible to have those two lines* removed from the manifest of the library? This should resolve the issue for those with different settings without causing any new issues afaik.

Thanks

*android:allowBackup=“true”
android:supportsRtl=“true”

**Comment by lordplagus02** [comment]
+1 experiencing this too

**Comment by JTBrinkmann** [comment]
This was "fixed" in 1.0.93. (which however isn't released, see [Jitpack](https://jitpack.io/#com.github.myinnos/AlphabetIndex-Fast-Scroll-RecyclerView), somewhat related to #48)

A **workaround** is to add `tools:replace="android:label,android:allowBackup,android:supportsRtl"` to the `<application>` tag in your application's AndroidManifest.xml (**not** the `<manifest>` tag).

A longterm solution would be appreciated though @myinnos
